tokens.tf

```diff
@@ -0,0 +1,49 @@
+#
+# @NOTE: At the moment we use the token webhook bridge to manage our user tokens, but we need a user to manage
+# tokens.csv content; this will be done by the platform hub. This user has to be managed for obvious reasons.
+# Hopefully once we've come up with a better implementation we can drop this requirement. Note, the following is
+# ONLY done if the module is managing the state bucket
+#
+
+# The policy used by the assets user
+data "aws_iam_policy_document" "assets_policy" {
+  statement {
+    actions = ["s3:*"]
+
+    resources = [
+      "arn:aws:s3:::${var.kops_state_bucket}/${var.environment}.${var.dns_zone}-assets/*",
```

Comment:
this is basically to facilitate a hack on how we manage tokens, which will hopefully go away .. I made the name generic just in case, but ultimately nothing but the tokens.csv should live here ..